protocol/messenger_messages_order_controller_test.go

```diff
@@ -0,0 +1,126 @@
+package protocol
+
+import (
+	"bytes"
+	"sort"
+	"sync"
+
+	"github.com/status-im/status-go/eth-node/types"
+	"github.com/status-im/status-go/protocol/transport"
+)
+
+type messagesOrderType int
+
+const (
+	messagesOrderRandom messagesOrderType = iota
+	messagesOrderInPostOrder
```

Comment:
duplication the word "order" is a bit confusing here

```suggestion
	messagesOrderAsPosted
```